dubbo-remoting/dubbo-remoting-etcd3/src/main/java/org/apache/dubbo/remoting/etcd/jetcd/JEtcdClientWrapper.java

```diff
@@ -488,9 +487,9 @@ public void delete(String path) {
     public String[] endPoints(String backupAddress) {
         String[] endpoints = backupAddress.split(COMMA_SEPARATOR);
         List<String> addresses = Arrays.stream(endpoints)
-                .map(address -> address.contains(Constants.HTTP_SUBFIX_KEY)
+                .map(address -> address.contains(org.apache.dubbo.remoting.etcd.Constants.HTTP_SUBFIX_KEY)
```

Comment:
This is best to `import static org.apache.dubbo.remoting.etcd.Constants.HTTP_SUBFIX_KEY` directly